src/lib/index.ts

```diff
@@ -139,6 +139,7 @@ export * from './utils';
 export * from './view-switcher';
 export * from './icon';
 export * from './stack';
+export * from './button-area';
```

Comment:
We also want to import the `ButtonAreaComponent` class and add it to the `CUSTOM_ELEMENTS` array below.